connection_test.go

```diff
@@ -0,0 +1,102 @@
+package ts3
+
+import (
+	"bufio"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/ssh"
+)
+
+var sshClientTestConfig = &ssh.ClientConfig{HostKeyCallback: ssh.InsecureIgnoreHostKey()} //nolint:gosec
```

Comment:
Style nit
```suggestion
var (
	sshClientTestConfig = &ssh.ClientConfig{
		HostKeyCallback: ssh.InsecureIgnoreHostKey() //nolint:gosec
	}
)
```

Reply:
First of, thanks for your review. :)

While I don't mind writing more lines I do not agree with using a `var ()` block with a single variable.

Reply:
That's a standard we use, but it's not big deal so we can skip that one ;-)

Reply:
`gofumpt` also agrees with you so I guess that's something we need to look at adopting.